storage/interceptors/broker_catalog_helper.go

```diff
@@ -0,0 +1,73 @@
+package interceptors
+
+import (
+	"context"
+	"fmt"
+	"github.com/Peripli/service-manager/pkg/instance_sharing"
+	"github.com/Peripli/service-manager/pkg/query"
+	"github.com/Peripli/service-manager/pkg/types"
+	"github.com/Peripli/service-manager/storage"
+	"github.com/gofrs/uuid"
+	"strconv"
+	"time"
+)
+
+func convertReferencePlanObjectToOSBPlan(plan *types.ServicePlan) interface{} {
+	return map[string]interface{}{
+		"id":          plan.ID,
+		"name":        plan.Name,
+		"description": plan.Description,
+		"bindable":    plan.Bindable,
+	}
+}
+
+func generateReferencePlanObject(serviceOfferingId string) *types.ServicePlan {
+	referencePlan := new(types.ServicePlan)
+
+	UUID, err := uuid.NewV4()
+	if err != nil {
+		panic(fmt.Errorf("could not generate GUID for ServicePlan: %s", err))
+	}
+
+	referencePlan.ID = UUID.String()
+	referencePlan.CatalogID = UUID.String()
+	referencePlan.CatalogName = instance_sharing.ReferencePlanName
+	referencePlan.Name = instance_sharing.ReferencePlanName
+	referencePlan.Description = instance_sharing.ReferencePlanDescription
+	referencePlan.ServiceOfferingID = serviceOfferingId
+	referencePlan.Bindable = newTrue()
+	referencePlan.Ready = true
+	referencePlan.CreatedAt = time.Now()
+	referencePlan.UpdatedAt = time.Now()
+
+	return referencePlan
+}
+
+func isBindablePlan(service *types.ServiceOffering, plan *types.ServicePlan) bool {
+	if plan.Bindable != nil {
+		return *plan.Bindable
+	}
+	return service.Bindable
+}
+
```

Comment:
we tried it before - moving the bindable to the same place of the `IsShareablePlan()`
but since it is under the `service_plan_utils.go` file, we cannot do that because we also check the service-offering.